frontend/packages/console-shared/src/selectors/common.ts

```diff
@@ -10,3 +10,6 @@ export const getUid = (value: K8sResourceKind) =>
   _.get(value, 'metadata.uid') as K8sResourceKind['metadata']['uid'];
 export const getDeletetionTimestamp = (value: K8sResourceKind) =>
   _.get(value, 'metadata.deletionTimestamp') as K8sResourceKind['metadata']['deletionTimestamp'];
+
+export const getStatusPhase = (value: K8sResourceKind) =>
```

Comment:
This seems like a dubious helper because only certain kinds have `status.phase`. It would be better to add helpers for specific types with specific enumerated return values.